haproxy/datadog_checks/haproxy/haproxy.py

```diff
@@ -79,7 +79,9 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         "eresp": ("rate", "errors.resp_rate"),
         "wretr": ("rate", "warnings.retr_rate"),
         "wredis": ("rate", "warnings.redis_rate"),
+        "lastchg": ("gauge", "uptime"),
         "req_rate": ("gauge", "requests.rate"),  # HA Proxy 1.4 and higher
+        "req_tot": ("rate", "requests.tot_rate"),  # HA Proxy 1.4 and higher
```

Comment:
We're a bit inconsistent with plural there... But I think it should be `total`. It's not a `rate`, is it?

Reply:
req_tot is a monotonically increasing value. Having it as a rate allows to display the total number of requests per second.
Suffixing `_rate` like in other haproxy metrics makes it clear that this metric is a rate. I've done for for consistency actually, I would have gone with `requests.total` otherwise